bgpd/bgp_bmp.c

```diff
@@ -2231,7 +2473,9 @@ DEFPY(bmp_monitor_cfg,
 	argv_find_and_parse_afi(argv, argc, &index, &afi);
 	argv_find_and_parse_safi(argv, argc, &index, &safi);
 
-	if (policy[1] == 'r')
+	if (policy[0] == 'l')
```

Comment:
can we use something other than "1" here? It's going to be hard for future readers to understand what this does ... maybe a `#define` or some such? ... also, is it possible there might be other policies than `1` here?

Reply:
this is an L not a 1 :) silly github font
this check matches :
* "p_**r**_e-policy"
* "p_**o**_st-policy"
* "_**l**_ocal-rib"

yeah maybe a #define POLICY_IS_PRE/POST/LOCRIB(..) could be nice